app/src/main/java/org/mozilla/focus/settings/SettingsFragment.java

```diff
@@ -154,7 +154,6 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
             }
             TelemetryWrapper.settingsLocaleChangeEvent(key, String.valueOf(locale), TextUtils.isEmpty(value));
             localeManager.updateConfiguration(getActivity(), locale);
-            FirebaseHelper.onLocaleUpdate(getActivity());
```

Comment:
we don't need to reset the default value after changing the locale now since remote config will always on